pkg/terraform/attribute.go

```diff
@@ -483,6 +482,60 @@ func (a *Attribute) Contains(checkValue interface{}, equalityOptions ...Equality
 	return strings.Contains(val.AsString(), stringToLookFor)
 }
 
+func (a *Attribute) OnlyContains(checkValue interface{}) bool {
```

Comment:
We could do `checkValue ...interface{}` so you can do `OnlyContains(x)` or `OnlyContains(x, y, z)` - WDYT?

Reply:
(then we can remove the check for `checkValue.([]interface{})` on line 494 too